src/cli-util.js

```diff
@@ -299,7 +299,14 @@ function eachFilename(argv, patterns, callback) {
       return;
     }
     ignorer
-      .filter(filePaths)
+      .filter(
+        filePaths.map(
+          filePath =>
+            filePath.startsWith(process.cwd())
```

Comment:
This will work for `/Users/lj/test/fixtures/error.js`, but fail for `/Users/lj/test/./fixtures///error.js`. I think just doing `filePaths.map(filePath => path.relative(process.cwd(), filePath))` should do what we want in all cases.